python/tvm/driver/tvmc/common.py

```diff
@@ -133,7 +133,7 @@ def tokenize_target(target):
 
     target_pattern = (
         r"(\-{0,2}[\w\-]+\=?"
-        r"(?:[\w\+\-]+(?:,[\w\+\-])*|[\'][\w\+\-,\s]+[\']|[\"][\w\+\-,\s]+[\"])*|,)"
+        r"(?:[\w\+\-\.]+(?:,[\w\+\-\.])*|[\'][\w\+\-,\s\.]+[\']|[\"][\w\+\-,\s\.]+[\"])*|,)"
```

Comment:
Would it be possible to comment what this regexp is doing?

Reply:
Agree. This regex is super complex so it would be better to comment what's doing.

Reply:
OK, just for the sake of change tracking, can I put a change in a new PR, splitting this regex into a series of simpler to understand chunks, and explaining each of them?

Reply:
Sounds good. Then I'll merge this PR first and you could incopreate the changes of unit tests to the follow-up PR as well.

Reply:
#7654